src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
here we need to import constant `BASE_URL` from `src/api/api.ts `to avoid duplicating the same variables in project (DRY principle)
check plz all cases with such duplicates